libraries/AC_CustomControl/AC_CustomControl.h

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
+/// @file    AC_CustomControl.h
+/// @brief   ArduCopter custom control library
+
+#include <AP_Common/AP_Common.h>
+#include <AP_Param/AP_Param.h>
+#include <AP_Vehicle/AP_Vehicle.h>
+#include <AP_AHRS/AP_AHRS_View.h>
+#include <AC_AttitudeControl/AC_AttitudeControl_Multi.h>
+#include <AP_Motors/AP_Motors.h>
+#include <AP_Logger/AP_Logger.h>
+
+#ifndef CUSTOMCONTROL_ENABLED
+#define CUSTOMCONTROL_ENABLED (!HAL_MINIMIZE_FEATURES && !defined(HAL_BUILD_AP_PERIPH) && BOARD_FLASH_SIZE > 1024 || CONFIG_HAL_BOARD == HAL_BOARD_SITL)
```

Comment:
should be compiled into SITL by default, but not real boards, but should have a ./waf configure --enable-custom-controller option to make it easy to enable on any board (see opendroneid-enable as an example)

Reply:
Added --enable-custom-controller option. Also forced CUSTOMCONTROL_ENABLED to 0 for Heli in config.h.